src/Components/Artist/MarketInsights.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface MarketInsightsProps extends RelayProps, React.HTMLProps<MarketInsights> {}
+
+const Categories = {
+  "blue-chip": "Blue Chip",
+  "top-established": "Top Established",
+  "top-emerging": "Top Emerging",
+}
+
+export class MarketInsights extends React.Component<MarketInsightsProps, null> {
+  renderGalleryRepresentation() {
+    const { highlights } = this.props.artist
+    const { partner_artists } = highlights
+    if (partner_artists && partner_artists.length > 0) {
+      const partnerArtist = partner_artists[0]
+      const { categories } = partnerArtist.partner
+      let category
+      Object.keys(Categories).forEach(key => {
```

Comment:
I can probably write this better, essentially this tries to inspect the embedded partner categories, to figure out which of the 3 special types we have.

Reply:
In practice I think we'll need to apply a hierarchy of preferences, e.g., to say "represented by blue chip galleries..." if there are any in that category, then "...top established..." if there are any in that category, and so on.